lib/matplotlib/colors.py

```diff
@@ -1493,6 +1493,11 @@ def hsv_to_rgb(hsv):
     return rgb
 
 
+# np.linalg.norm doesn't broadcast in numpy 1.7
+def _linalg_norm(arr, keepdims=False, axis=None):
+    return np.sqrt(np.sum(np.square(arr), axis=axis, keepdims=keepdims))
```

Comment:
Appear to also use it https://github.com/matplotlib/matplotlib/blob/master/lib/mpl_toolkits/mplot3d/axes3d.py#L2711